pkg/console/controllers/clidownloads/controller.go

```diff
@@ -188,6 +195,41 @@ The oc binary offers the same capabilities as the kubectl binary, but it is furt
 	}
 }
 
+func LicenseDownloads(host, cliDownloadsName string) *v1.ConsoleCLIDownload {
+	baseURL := fmt.Sprintf("%s", util.HTTPS(host))
+	platforms := []struct {
+		label    string
+		key      string
+		archType string
+	}{
+		{"Linux for x86_64", "amd64/linux", "LICENSE"},
+		{"Linux for ARM 64", "arm64/linux", "LICENSE"},
```

Comment:
```suggestion
		{"Linux for ARM 64 (unsupported)", "arm64/linux", "LICENSE"},
```

to be consistent with the `oc` binaries ?